### PR TITLE
disabled scrolling on popup

### PR DESCRIPTION
**Pull request by pavankalyan-5**

### CODE REVIEW

File: front-end/src/components/LoginPopup/LoginPopup.jsx [hunk 1]
```diff
@@ -51,7 +51,19 @@ const LoginPopup = ({setShowLogin}) => {
             password: ''
         })
         setErrorMessage(null)
-    },[currState])
+
+        // if showLogin is true disable scrolling
```
Comment (maheshvarma2001) [inline front-end/src/components/LoginPopup/LoginPopup.jsx:55]: Is this comment useful in testing

Comment (pavankalyan-5) [inline front-end/src/components/LoginPopup/LoginPopup.jsx:55]: yes

**Review by maheshvarma2001 — APPROVED** [review]
Approved

**Review by maheshvarma2001 — CHANGES_REQUESTED** [review]
Approved

**Review by maheshvarma2001 — APPROVED** [review]
# Approved

**Pull request merged**